README.md

```diff
@@ -3,65 +3,84 @@
 [![CircleCI](https://circleci.com/gh/numerai/numerai-cli.svg?style=svg)](https://circleci.com/gh/numerai/numerai-cli)
 [![PyPI](https://img.shields.io/pypi/v/numerai-cli.svg?color=brightgreen)](https://pypi.org/project/numerai-cli/)
 
-This is a CLI for setting up a Numerai compute node and deploying your models to it. This sets up a compute cluster in AWS (Amazon Web Services), and is architected to cost a minimal amount of money to run (on average, you will spend less than $1 per month).
+CLI to set up a Numerai Compute node in AWS (Amazon Web Services) and deploy your models to it. 
+This is architected to cost a minimal amount of money to run (less than $5 per month on average).
```

Comment:
This solution is architected to cost less and $5/mo on average (actual costs may vary).